docs/sources/next/javascript-api/k6-experimental/browser/response/json.md

```diff
@@ -0,0 +1,14 @@
+---
+title: 'json()'
+description: 'Browser module: Response.json method'
+---
+
+# json()
+
+Returns the JSON representation of response body. Throws if response body is not parsable via `JSON.parse`.
+
+### Returns
+
+| Type           | Description                           |
+| -------------- | ------------------------------------- |
+| `Promise<any>` | JSON representation of response body. |
```

Comment:
Avoided the example since I couldn't find a suitable `goto` navigation that would respond with a `json`.

Reply:
What about https://httpbin.org/#/Response_formats/get_json?

Reply:
I wanted to use a website that we own so that we have more control over it. Didn't feel right to use someone else's.

Reply:
httpbin.org is extensively used in k6-docs. [See this](https://github.com/search?q=repo%3Agrafana%2Fk6-docs%20httpbin&type=code). But I'm also happy without this change.

Reply:
There is also https://httpbin.test.k6.io that we own :)

Reply:
Good spot. I've added a fixup in c8906514b8bd5af00470993e4a93dec05897caaf